ksql-serde/src/main/java/io/confluent/ksql/serde/DataSource.java

```diff
@@ -22,9 +22,9 @@
 
   enum DataSourceSerDe { JSON, AVRO, DELIMITED }
 
-  final String AVRO_SERDE_NAME = "AVRO";
-  final String JSON_SERDE_NAME = "JSON";
-  final String DELIMITED_SERDE_NAME = "DELIMITED";
+  String AVRO_SERDE_NAME = "AVRO";
```

Comment:
why not `final`?

Reply:
modifier 'final' is redundant for interface fields

Reply:
sure - didn't realise it was an interface